workflow/alerting/alert-practices.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Common Practices - Alerts
+sidebarTitle: Common Practices
+---
+
+The following guide describes best practices for configuring, maintaining, and managing essential alerts. Let this be the starting point to ensure you are effectively monitoring your system-wide infrastructure data.
+
+# Resource Monitoring Alerts
+
+### CPU Utilization Alert
+
+The following alert tracks the average CPU Utilization for a Kubernetes node. This alert is monitoring a single Kubernetes node, specified by the Filter configuration. A Warning alert triggers whenever the average CPU Utilization surpasses three CPUs on the specified Kubernetes node, while a Critical alert activates when the average usage reaches five CPUs. This is checked on a five minute interval.
+
+![](https://lh7-us.googleusercontent.com/-IOXG6PBIGm_Ulzp7OS_D4s0TGhLoo_VQTI29xxG6bxHyHVY-F3JCQVXwcO8xn20Klkt6QaVQMTpJtdD461FfMIiScQuUagVs5un5RB_4ce1_vb45RbsxtzOg2U4cNwOQzN1HIBpzcg8tHZCHK-EQNU)
+
+### Memory Consumption Alert
+
+The following alert tracks the average amount of memory used when data is being read and written across your entire system. A Warning is fired when `system.memory.usage` is greater than 50 bytes and a Critical warning when it is greater than 75 bytes. This is checked on a five minute interval.
+
+<Note> The __State__ filter is left blank so that all state metrics (e.g. buffered, cached, free, etc.) are collected.</Note>
+
+![](https://lh7-us.googleusercontent.com/sjU54Mqe3XINAZ7_It_i01xLFQeqFIAdAFMN9t5ElSKz_5lOTYwz8ltjlt5g6g2dfnSeXgUF9PgwNGAXd0fEx2oUP0Cz-SsrFuSMr-_MsqjvGVuDXYB1TBQQm-HvdHx7bEM_v6n6N9b-xqwLg4zOIWE)
+
+### APM Monitoring Alert
+
+The following alert tracks the total number of trace requests from a single APM. This alert uses the `IN` operator to monitor a specific `service.name` with the Filter configuration. A Critical message is fired when the total number of trace requests exceeds 5,000 in a 10 minute period.
+
+![](https://lh7-us.googleusercontent.com/JcY5c276b6jpVekZJvWeo6QeM8M6So7CCfbbNDHcTi-VmNGa-crrgNd9MZSfgXTh3JUc6o8L6nkPxv5wNnuP9dVC8OOQVV_uQ-Ccqp7F4v6zpgmGmmH589dT8Y-QIlc0l4oCsoDW4rxuN8xZRA3V3-Q)
+
```

Comment:
Let's add one more about checking logs for certain strings, e.g. ones with the word "Error" in them.